lib/inline_svg/transform_pipeline/transformations.rb

```diff
@@ -25,12 +25,25 @@ def self.all_transformations
 
   def self.lookup(transform_params)
     without_empty_values(transform_params).map do |key, value|
```

Comment:
I agree the method's name is misleading, but `without_empty_values` and `all_default_values ` in this case are maps not arrays, I'll change it to use another method with a better meaning